examples/gpio_timer.rs

```diff
@@ -3,7 +3,11 @@
 
 extern crate panic_rtt_target;
 
-use lpc8xx_hal::{cortex_m_rt::entry, gpio::Level, prelude::*, Peripherals};
+use core::convert::TryInto as _;
```

Comment:
Any reason why it's imported as hidden? I don't see why this shouldn't be in the local namespace. 

If this is just your stylistic preference, that's also fine

Reply:
I always do this when I need a trait because I'm using its methods, but don't refer to it by name. Makes it a bit more obvious why it's imported, and keeps the local namespace clean.

But it's just a small thing, not important at all. Happy to change it or leave it, however you prefer.